config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/customers/:id", to: "customers#show"
+  post "/customers/:id", to: "customer_items#create"
```

Comment:
See comment above -- consider `post "/customers/:id/items", to: "customer_items#create"`